iot_verifier/src/poc_report.rs

```diff
@@ -58,6 +58,7 @@ pub struct Report {
     pub created_at: Option<DateTime<Utc>>,
     #[sqlx(default)]
     pub timestamp: Option<DateTime<Utc>>,
+    pub version: Option<i32>,
```

Comment:
should this also be `entropy_version` so it's clear it refers to the version of the entropy struct and not the version of the Report struct?

Reply:
agree it should and same for timestamp.  but i would have to use serde rename and then backfill the changes.  Can follow that up with a diff PR.